src/app/features/landing/components/landing-footer/landing-footer.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { LayoutService } from 'src/app/core/layout/service/app.layout.service';
+
+@Component({
+  selector: 'landing-footer',
+  templateUrl: './landing-footer.component.html',
+  styleUrls: ['./landing-footer.component.scss'],
+})
+export default class LandingFooterComponent {
+  constructor(
+    public layoutService: LayoutService,
+    public router: Router,
+  ) {
+    console.log('constructor logic here');
```

Comment:
Run lint:fix, console logs should not be present.

Reply:
Yes agree. But we have a rule for angular which do not allow empty constructors. So had to add dummy logs here.

Reply:
Thats fine, you can add the disable rule for constructor instead of console.log.

Reply:
Alright, @xixas  I have added inline disable comments and removed console logs in constructors.